src/components/ContactDetails.vue

```diff
@@ -570,7 +624,35 @@ export default {
 
 			return ''
 		},
+		profilePageLink() {
+			// just for testing this is not fully correct
+			if (this.profileEnabled) {
+				return generateUrl('/u/{userId}', { userId: getCurrentUser().uid })
+			}
+			return null
+		},
+		emailAddressProperty() {
```

Comment:
```suggestion
		emailAddressProperties() {
```

it's none, one or more